<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="5" ignored="0" total="28" passed="23">
  <reporter-output>
  </reporter-output>
  <suite name="All Test Suite" duration-ms="34444" started-at="2019-08-25T13:56:02Z" finished-at="2019-08-25T13:56:36Z">
    <groups>
    </groups>
    <test name="E:\Selenium\E2EAPITesting" duration-ms="34444" started-at="2019-08-25T13:56:02Z" finished-at="2019-08-25T13:56:36Z">
      <class name="com.employeeapi.testcases.Post_Employee_Record">
        <test-method status="PASS" signature="CreateEmployee()[pri:0, instance:com.employeeapi.testcases.Post_Employee_Record@7ca48474]" name="CreateEmployee" is-config="true" duration-ms="5319" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateEmployee -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.employeeapi.testcases.Post_Employee_Record@7ca48474]" name="setup" is-config="true" duration-ms="0" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="CheckContentType()[pri:0, instance:com.employeeapi.testcases.Post_Employee_Record@7ca48474]" name="CheckContentType" duration-ms="1" started-at="2019-08-25T13:56:24Z" finished-at="2019-08-25T13:56:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentType -->
        <test-method status="FAIL" signature="CheckResponseBody()[pri:0, instance:com.employeeapi.testcases.Post_Employee_Record@7ca48474]" name="CheckResponseBody" duration-ms="2" started-at="2019-08-25T13:56:24Z" finished-at="2019-08-25T13:56:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.employeeapi.testcases.Post_Employee_Record.CheckResponseBody(Post_Employee_Record.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckResponseBody -->
        <test-method status="PASS" signature="CheckStatusCode()[pri:0, instance:com.employeeapi.testcases.Post_Employee_Record@7ca48474]" name="CheckStatusCode" duration-ms="1" started-at="2019-08-25T13:56:24Z" finished-at="2019-08-25T13:56:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusCode -->
        <test-method status="PASS" signature="CheckStatusLine()[pri:0, instance:com.employeeapi.testcases.Post_Employee_Record@7ca48474]" name="CheckStatusLine" duration-ms="1" started-at="2019-08-25T13:56:24Z" finished-at="2019-08-25T13:56:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusLine -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.employeeapi.testcases.Post_Employee_Record@7ca48474]" name="teardown" is-config="true" duration-ms="2" started-at="2019-08-25T13:56:24Z" finished-at="2019-08-25T13:56:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.testcases.Post_Employee_Record -->
      <class name="com.employeeapi.testcases.Get_Single_Employee_Record">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="setup" is-config="true" duration-ms="3" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="getEmployeeData()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="getEmployeeData" is-config="true" duration-ms="3278" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployeeData -->
        <test-method status="PASS" signature="CheckContentLength()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="CheckContentLength" duration-ms="2" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentLength -->
        <test-method status="PASS" signature="CheckContentType()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="CheckContentType" duration-ms="2" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentType -->
        <test-method status="FAIL" signature="CheckResponseBody()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="CheckResponseBody" duration-ms="3" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.employeeapi.testcases.Get_Single_Employee_Record.CheckResponseBody(Get_Single_Employee_Record.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckResponseBody -->
        <test-method status="PASS" signature="CheckResponseTime()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="CheckResponseTime" duration-ms="3" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckResponseTime -->
        <test-method status="FAIL" signature="CheckServerType()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="CheckServerType" duration-ms="2" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [nginx/1.14.1] but found [Apache]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [Apache]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.employeeapi.testcases.Get_Single_Employee_Record.CheckServerType(Get_Single_Employee_Record.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckServerType -->
        <test-method status="PASS" signature="CheckStatusCode()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="CheckStatusCode" duration-ms="3" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusCode -->
        <test-method status="PASS" signature="CheckStatusLine()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="CheckStatusLine" duration-ms="1" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusLine -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.employeeapi.testcases.Get_Single_Employee_Record@13a57a3b]" name="teardown" is-config="true" duration-ms="0" started-at="2019-08-25T13:56:19Z" finished-at="2019-08-25T13:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.testcases.Get_Single_Employee_Record -->
      <class name="com.employeeapi.testcases.Put_Employee_Record">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.employeeapi.testcases.Put_Employee_Record@337d0578]" name="setup" is-config="true" duration-ms="4" started-at="2019-08-25T13:56:24Z" finished-at="2019-08-25T13:56:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="CreateEmployee()[pri:0, instance:com.employeeapi.testcases.Put_Employee_Record@337d0578]" name="CreateEmployee" is-config="true" duration-ms="5322" started-at="2019-08-25T13:56:24Z" finished-at="2019-08-25T13:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateEmployee -->
        <test-method status="PASS" signature="CheckContentType()[pri:0, instance:com.employeeapi.testcases.Put_Employee_Record@337d0578]" name="CheckContentType" duration-ms="1" started-at="2019-08-25T13:56:30Z" finished-at="2019-08-25T13:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentType -->
        <test-method status="PASS" signature="CheckResponseBody()[pri:0, instance:com.employeeapi.testcases.Put_Employee_Record@337d0578]" name="CheckResponseBody" duration-ms="2" started-at="2019-08-25T13:56:30Z" finished-at="2019-08-25T13:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckResponseBody -->
        <test-method status="PASS" signature="CheckStatusCode()[pri:0, instance:com.employeeapi.testcases.Put_Employee_Record@337d0578]" name="CheckStatusCode" duration-ms="1" started-at="2019-08-25T13:56:30Z" finished-at="2019-08-25T13:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusCode -->
        <test-method status="PASS" signature="CheckStatusLine()[pri:0, instance:com.employeeapi.testcases.Put_Employee_Record@337d0578]" name="CheckStatusLine" duration-ms="1" started-at="2019-08-25T13:56:30Z" finished-at="2019-08-25T13:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusLine -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.employeeapi.testcases.Put_Employee_Record@337d0578]" name="teardown" is-config="true" duration-ms="1" started-at="2019-08-25T13:56:30Z" finished-at="2019-08-25T13:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.testcases.Put_Employee_Record -->
      <class name="com.employeeapi.testcases.Delete_Employee_Record">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.employeeapi.testcases.Delete_Employee_Record@59e84876]" name="setup" is-config="true" duration-ms="8" started-at="2019-08-25T13:56:30Z" finished-at="2019-08-25T13:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="DeleteEmployee()[pri:0, instance:com.employeeapi.testcases.Delete_Employee_Record@59e84876]" name="DeleteEmployee" is-config="true" duration-ms="6513" started-at="2019-08-25T13:56:30Z" finished-at="2019-08-25T13:56:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeleteEmployee -->
        <test-method status="PASS" signature="CheckContentType()[pri:0, instance:com.employeeapi.testcases.Delete_Employee_Record@59e84876]" name="CheckContentType" duration-ms="0" started-at="2019-08-25T13:56:36Z" finished-at="2019-08-25T13:56:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentType -->
        <test-method status="PASS" signature="CheckResponseBody()[pri:0, instance:com.employeeapi.testcases.Delete_Employee_Record@59e84876]" name="CheckResponseBody" duration-ms="1" started-at="2019-08-25T13:56:36Z" finished-at="2019-08-25T13:56:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckResponseBody -->
        <test-method status="PASS" signature="CheckStatusCode()[pri:0, instance:com.employeeapi.testcases.Delete_Employee_Record@59e84876]" name="CheckStatusCode" duration-ms="1" started-at="2019-08-25T13:56:36Z" finished-at="2019-08-25T13:56:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusCode -->
        <test-method status="PASS" signature="CheckStatusLine()[pri:0, instance:com.employeeapi.testcases.Delete_Employee_Record@59e84876]" name="CheckStatusLine" duration-ms="0" started-at="2019-08-25T13:56:36Z" finished-at="2019-08-25T13:56:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusLine -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.employeeapi.testcases.Delete_Employee_Record@59e84876]" name="teardown" is-config="true" duration-ms="2" started-at="2019-08-25T13:56:36Z" finished-at="2019-08-25T13:56:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.testcases.Delete_Employee_Record -->
      <class name="com.employeeapi.testcases.Get_All_Employees">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="setup" is-config="true" duration-ms="42" started-at="2019-08-25T13:56:02Z" finished-at="2019-08-25T13:56:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="getAllEmployees()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="getAllEmployees" is-config="true" duration-ms="13533" started-at="2019-08-25T13:56:02Z" finished-at="2019-08-25T13:56:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllEmployees -->
        <test-method status="PASS" signature="CheckContentEncoding()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckContentEncoding" duration-ms="5" started-at="2019-08-25T13:56:15Z" finished-at="2019-08-25T13:56:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentEncoding -->
        <test-method status="PASS" signature="CheckContentLength()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckContentLength" duration-ms="1" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentLength -->
        <test-method status="PASS" signature="CheckContentType()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckContentType" duration-ms="1" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckContentType -->
        <test-method status="PASS" signature="CheckCookie()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckCookie" duration-ms="2" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckCookie -->
        <test-method status="PASS" signature="CheckResponseBody()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckResponseBody" duration-ms="44" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckResponseBody -->
        <test-method status="FAIL" signature="CheckResponseTime()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckResponseTime" duration-ms="75" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.employeeapi.testcases.Get_All_Employees.CheckResponseTime(Get_All_Employees.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckResponseTime -->
        <test-method status="FAIL" signature="CheckServerType()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckServerType" duration-ms="3" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [nginx/1.14.1] but found [Apache]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [Apache]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.employeeapi.testcases.Get_All_Employees.CheckServerType(Get_All_Employees.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckServerType -->
        <test-method status="PASS" signature="CheckStatusCode()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckStatusCode" duration-ms="1" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusCode -->
        <test-method status="PASS" signature="CheckStatusLine()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="CheckStatusLine" duration-ms="0" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckStatusLine -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.employeeapi.testcases.Get_All_Employees@69d9c55]" name="teardown" is-config="true" duration-ms="1" started-at="2019-08-25T13:56:16Z" finished-at="2019-08-25T13:56:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.employeeapi.testcases.Get_All_Employees -->
    </test> <!-- E:\Selenium\E2EAPITesting -->
  </suite> <!-- All Test Suite -->
</testng-results>
